man/usr/share/man/man1m/piadm.1m.md

```diff
@@ -69,6 +70,15 @@ piadm(1M) -- Manage SmartOS Platform Images
     itself without issue.  Occasionally, however, a PI will have Boot Image
     changes also that need to accompany it.
 
+## iPXE
+
+    On Triton Compute Nodes, which run SmartOS booted off a network, stored
+    locally is an implementation of iPXE, an enhanced form of Preboot
+    eXecution Environment (PXE). piadm(1M) can install iPXE in lieu of the
+    normal Boot Image and Platform Image.  The word "ipxe" can be used in
+    place of a PI stamp or a source.  Unlike Platform Images, ipxe can only
+    have one instance maintained on a given bootable pool.
```

Comment:
For clarity, I think it should also state that the version of ipxe is controlled by the Triton headnode, and piadm will simply put the provided ipxe in place, and possibly refer to the sdcadm man page.